src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fuzzing engine interface."""
+
+from bot.fuzzers import engine
+from bot.fuzzers import engine_common
+from bot.fuzzers import libfuzzer
```

Comment:
Remove this unnecessary import. You can do a `python butler.py lint` to lint this PR.

Reply:
This version is outdated. The import is removed and the commit did pass lint.